cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -279,6 +282,11 @@ public void execute() {
             return;
         }
 
+        if (workers < 1) {
+            this.outStream.println("\nWarning: Workers can not be negative or zero. Test execution is proceeded " +
+                    "with default worker count.\n");
```

Comment:
Wouldn't it be better to mention what our default value is here. 
Perhaps we can move `defaultValue` to a constant and add here.
```suggestion
                    "with default worker count (" + DEFAULT_WORKER_COUNT + ").\n");
```

Reply:
worker related options are no longer valid